pkg/controller/reconciler.go

```diff
@@ -85,16 +80,27 @@ func doReconcile(kc *kubeletClient, driver *PodDriver) {
 			if value, ok := pod.Labels[config.PodTypeKey]; !ok || value != config.PodTypeValue {
 				continue
 			}
-			err := func() error {
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
 				ctx, cancel := context.WithTimeout(context.Background(), config.ReconcileTimeout)
 				defer cancel()
-				return driver.Run(ctx, pod)
+				mounter := mount.SafeFormatAndMount{
+					Interface: mount.New(""),
+					Exec:      k8sexec.New(),
+				}
+
+				driver := NewPodDriver(ks, mounter)
```

Comment:
Why do we need an independent driver for each Pod?

Reply:
podDriver handlers is a map and select a handler by pod status. If all pods goroutine use one driver may result in `concurrent map writes map`. Any better idea?

Reply:
Ok, looks good to me.